kornia/color/colormap.py

```diff
@@ -1,126 +1,121 @@
 from abc import ABC
-from typing import List, Optional
+from typing import List, Optional, Union
 
 import torch
 from torch.nn.functional import interpolate
 
+from kornia.color import colormap_data as cm_data
 from kornia.core import Module, Tensor, tensor
 from kornia.core.check import KORNIA_CHECK_IS_GRAY
 
-RGBColor = List[float]
+RGBColor = cm_data.RGBColor
+
+colormap_options = {
+    "autumn": cm_data._AUTUMN_BASE,
+    "bone": cm_data._BONE_BASE,
+    "jet": cm_data._JET_BASE,
+    "winter": cm_data._WINTER_BASE,
+    "rainbow": cm_data._RAINBOW_BASE,
+    "ocean": cm_data._OCEAN_BASE,
+    "summer": cm_data._SUMMER_BASE,
+    "spring": cm_data._SPRING_BASE,
+    "cool": cm_data._COOL_BASE,
+    "hsv": cm_data._HSV_BASE,
+    "brg": cm_data._BRG_BASE,
+    "pink": cm_data._PINK_BASE,
+    "hot": cm_data._HOT_BASE,
+    "plasma": cm_data._PLASMA_BASE,
+    "viridis": cm_data._VIRIDIS_BASE,
+    "cividis": cm_data._CIVIDIS_BASE,
+    "twilight": cm_data._TWILIGHT_BASE,
+    "turbo": cm_data._TURBO_BASE,
+    "seismic": cm_data._SEISMIC_BASE,
+}
```

Comment:
But, how each colormap data, should be in a `getter` function, I think is best just to put a bunch of ifs's here or something like that, otherwise will initialize all base data unnecessarily